misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt

```diff
@@ -100,6 +104,52 @@ abstract class PathFeatureAssertionsSpec(
         }
     }
 
+    describeFun("val ${fileNameFeature.name}") {
+        val fileNameVal = fileNameFeature.lambda
+
+        context("File with extension") {
+            it("toBe(my.txt) holds") {
+                expect(Paths.get("a/my.txt")).fileNameVal().toBe("my.txt")
+            }
+            it("toBe(my.txt) fails") {
+                expect {
+                    expect(Paths.get("a/my")).fileNameVal().toBe("my.txt")
+                }.toThrow<AssertionError> {
+                    messageContains("$fileNameDescr: \"my\"")
+                }
+            }
+        }
+    }
+
+    describeFun("fun ${fileName.name}") {
+        val fileNameFun = fileName.lambda
+
+        context("File with extension") {
```

Comment:
same here, rename the context
```suggestion
        context("path a/my.txt") {
```